eng/WindowsFormsImports.targets

```diff
@@ -0,0 +1,51 @@
+<Project>
```

Comment:
Since this file will only be used when packaging the WinForms props/targets into the WindowsDekstop SDK, could it be moved to the WindowsDesktop packaging folder and renamed?

eng/WindowsFormsImports.target -> packaging/Microsoft.NET.Sdk.WindowsDesktop/Microsoft.NET.Sdk.WindowsDesktop.WinForms.Packaging.targets?

Reply:
Actually, this could also go in eng\WpfArcadeSdk\tools alongside the other packaging targets.

Reply:
Putting Windows Forms specific targets in eng\WpfArcadeSdk\tools feels wrong. `packaging/Microsoft.NET.Sdk.WindowsDesktop/Microsoft.NET.Sdk.WindowsDesktop.WinForms.Packaging.targets` is probably a better suggestion.